code/_globalvars/lists/poll_ignore.dm

```diff
@@ -27,6 +27,13 @@
 #define POLL_IGNORE_LAVALAND_ELITE "lavaland_elite"
 #define POLL_IGNORE_SHUTTLE_DENIZENS "shuttle_denizens"
 #define POLL_IGNORE_BOTS "bots"
+#define POLL_IGNORE_HERETIC_MONSTER "heretic_monster"
+#define POLL_IGNORE_RAW_PROPHET "raw_prophet"
+#define POLL_IGNORE_STALKER "stalker"
+#define POLL_IGNORE_FIRE_SHARK "fire_shark"
+#define POLL_IGNORE_ASH_SPIRIT "ash_spirit"
+#define POLL_IGNORE_RUST_SPIRIT "rust_spirit"
+#define POLL_IGNORE_MAID_IN_MIRROR "maid_in_mirror"
```

Comment:
would you midn alphabetizing this and the list after it? it's definitely not the first time we've had PRs adding stuff to this, and it's not going to be the last. just helps maintainability and is cleaner for people in the future. thanks

Reply:
Sure thing